recipe/meta.yaml

```diff
@@ -49,7 +49,6 @@ test:
     - gemmi
   requires:
     - pip
-    - numpy
```

Comment:
So somewhat recently, numpy added itself a run_export, so it will now declare the versions of numpy it is compatible with, and automatically pull numpy in as a dependency to gemmi

if you don't want this, you will have to do more work (which I don't really recommend since it is error prone)